packages/cli/src/cmds/agentInstaller/agentProcedure.ts

```diff
@@ -80,7 +80,8 @@ export default abstract class AgentProcedure {
     const validationResult = JSON.parse(stdout);
 
     const errors = Array.isArray(validationResult) ? validationResult : validationResult.errors;
-    if (errors.length > 0) {
+    // if there were no errors then there's no "errors" key
+    if (errors && errors.length > 0) {
```

Comment:
Thanks, good catch!

That's what you get when you (@apotterri, I suppose) miss an implicit `any` :) Type checking would have caught that. I suppose it might be a good idea to add concrete types to this file and make sure it all typechecks one of these days.